src/components/Dashboard/FilingHistoryList/filings/ConsentContinuationOut.vue

```diff
@@ -45,6 +47,11 @@ export default class ConsentContinuationOut extends Vue {
   get hasEffectOfOrder (): boolean {
     return Boolean(this.filing.data?.order?.effectOfOrder)
   }
+
+  /** Whether the filing is complete. */
+  get isFilingComplete (): boolean {
+    return !EnumUtilities.isStatusPending(this.filing) && Boolean(this.filing.data)
```

Comment:
I don't think we check whether the data object exists anywhere else, so I think you should leave it out here also.

The numerous places in this component where we use conditional chaining (`this.filing.data?.something`) handles de-reference errors in case data is absent. I think that's enough.

What do you think?

Reply:
Yeah I wasn't too sure whether there were any other cases where a `filing` object might not have a `data` object, other than when the `status` is pending.

If that can't happen then it should be okay to remove the check on whether the data object exists.

Reply:
There may be other cases where a data object is absent -- that's why conditional chaining is used here.

Reply:
It looks like conditional chaining in the component was already catching the case where the `data` object is absent, and using "[unknown]" as the fallback for the expiry, leading to this bug. I'm not 100% sure if just removing the explicit check will work correctly, since the bug was already occurring with the conditional chaining working as expected. 

Although, perhaps there is a way to conditionally show the consent for continuation out `body` slot that does not require the explicit check to see if the `body` object exists, I am just not sure what that would look like.

Reply:
You now show body content only if the filing is complete, and I think we can assume the data is there in that case, so I think the bug is solved ...? (And you don't need to explicit check for the data object,,,?)

Reply:
I see, so would it make more sense then for me to use `EnumUtilities.IsStatusCompleted` rather than `!EnumUtilities.IsStatusPending`?

Reply:
Whoops, I didn't notice that. Yes, for sure, good catch! See how the other filings do it.

Reply:
Latest commit removes the explicit check of `data` and uses `isStatusCompleted`. Also includes both package files with updated version.